spaceship/lib/spaceship/connect_api/testflight/testflight.rb

```diff
@@ -412,6 +412,15 @@ def get_beta_tester_metrics(filter: {}, includes: nil, limit: nil, sort: nil)
           test_flight_request_client.get("betaTesterMetrics", params)
         end
 
+        #
+        # buildBundles
+        #
+
+        def get_build_bundle_build_bundle_file_sizes(build_bundle_id:, limit: nil)
+          params = test_flight_request_client.build_params(filter: nil, includes: nil, limit: limit, sort: nil, cursor: nil)
+          test_flight_request_client.get("buildBundles/#{build_bundle_id}/buildBundleFileSizes", params)
```

Comment:
Am I in the right place? Is the TestFlight client the right place to implement this kind of thing?